kuma/wiki/models.py

```diff
@@ -1358,22 +1322,10 @@ def redirect_url(self):
                 if len(url) > 1:
                     if url.startswith(settings.SITE_URL):
                         return url
-                    elif (url[0] == '/' and url[1] != '/'):
+                    elif url[0] == '/' and url[1] != '/':
                         return url
-                elif (len(url) == 1 and url[0] == '/'):
+                elif len(url) == 1 and url[0] == '/':
                     return url
-                else:
-                    return None
```

Comment:
We lost this `else` block that explicitly returned `None`? So the tests pass here because of an implicit return of `None` which feels kinda shaky for code that is meant to protect users against malicious redirects.